includes/MediaWiki/Api/EditEndpoint.php

```diff
@@ -182,10 +183,14 @@ public function run() {
 		$reconciliationDataValue = $reconciliationMainSnak->getDataValue();
 		$reconciliationUrl = $reconciliationDataValue->getValue();
 
-		$reconciliationItem = $this->reconciliationService->getOrCreateItemByStatementUrl(
-			$reconcileUrlProperty,
-			$reconciliationUrl
-		);
+		try {
```

Comment:
The call to `$this->minimalItemInput->getItem()` earlier (ca. line 140) can also throw a `ReconciliationException`, if I’m not mistaken, so I think this try-catch needs to be “widened” quite a bit (or add a separate one up there).

Alternatively, if #26 gets merged first, the try-catch only needs to wrap one line if I’m not mistaken:

```php
$request = $this->editRequestParser->parseRequestInterface( $this->getRequest() );
```

Reply:
Done.